tasks/section5.yml

```diff
@@ -399,7 +545,7 @@
       - rule_5.2.12
 
 - name: "SCORED | 5.2.13 | AUDIT | Ensure SSH Idle Timeout Interval is configured"
-  command: /bin/true
+  command: grep "^ClientAliveInterval" /etc/ssh/sshd_confige
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*